Source/AnimatableImageView.swift

```diff
@@ -28,10 +75,16 @@ public class AnimatableImageView: UIImageView {
   ///
   /// - parameter data: GIF image data.
   public func prepareForAnimation(imageData data: NSData) {
-    image = UIImage(data: data)
     animator = Animator(data: data, size: frame.size, contentMode: contentMode, framePreloadCount: framePreloadCount)
-    animator?.prepareFrames()
-    attachDisplayLink()
+    
+    if let animator = animator {
```

Comment:
Since all of the happy path code is inside the `if let` let's use `guard let animator = animator { return }` instead.